### PR TITLE
Add release notes for v0.7.3

### PR DESCRIPTION
**Pull request by mrsarm**
Release notes.

Issue: #148

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ The medic-android application is a thin wrapper to load the [CHT Core Framework]
 
 # Release notes
 
+### 0.7.3
+
```
Comment (garethbowen) [inline README.md:9]: Are there any breaking changes or upgrade steps people should know about? For example, what if they were using the permission before?

Should it be version 0.8.0 to make that more clear?

Comment (mrsarm) [reply README.md:9]: We only have removed the permission from the flavors we confirmed are not using the permission, while the "unbranded" flavor used for general testing still have the perm, despite it doesn't look to work as #127 points out. So no, it won't break any flavor.

Comment (garethbowen) [reply README.md:9]: Awesome. Thanks for clarifying!

**Review by garethbowen — APPROVED** [review]
LGTM!

**Pull request merged**